test/testLottie.cpp

```diff
@@ -70,6 +70,33 @@ TEST_CASE("Lottie Slot", "[tvgLottie]")
     REQUIRE(Initializer::term() == Result::Success);
 }
 
+TEST_CASE("Lottie Slot(Keyframe)", "[tvgLottie]")
```

Comment:
I think it's hard to identify the 'Keyframe' in the test suite itself. Let's add it as "Lottie Slot2".